README.md

```diff
@@ -1,20 +1,50 @@
 [![Documentation Status](https://readthedocs.org/projects/deepcell-kiosk-figure-generation/badge/?version=latest)](https://deepcell-kiosk-figure-generation.readthedocs.io/en/latest/?badge=latest)
 
-Recreate the figures from https://www.biorxiv.org/content/10.1101/505032v3 in 4 easy steps!
-
+The DeepCell Kiosk paper [(Bannon et al., 2020)](<https://www.biorxiv.org/content/10.1101/505032v3>) presents cost and runtime benchmarks for running a generic image segmentation pipeline on datasets of given sizes inside the DeepCell Kiosk with different sets of constraints (Fig. 1b).
+ 
 N.B.: Before following any of these steps, I'd recommend activating a Python `virtualenv`. It won't affect how easy it is to run this project, probably, but it will make your life easier after you've forgotten about this project and moved on to other Python packages.
```

Comment:
Let's rewrite this to remove the "I".

Reply:
+1 on the "I" (And this should be addressed throughout the README as it shows up in other places).

The bigger question is whether this paragraph is helpful. To get to this point, we assume you have enough programming knowledge to run the benchmarks pod. If you do, I think it's safe to assume you can either A) handle dependencies or B) Know about virtual environments. 

Perhaps we should mention here that you should have benchmarks to make figures and a link back to the developer docs. Otherwise, it may be helpful to move up the info from bullet point 2 to this location.

Reply:
Both comments implemented fully. This led to me reorganizing the README somewhat to make it follow the user's probable workflow (generate all the benchmarking data first, then start the figuremaking process) better.